packages/web/integrations/axelar/types.ts

```diff
@@ -71,40 +76,680 @@ export type SourceChainConfig = {
 export const SourceChainConfigs: {
   [asset: string]: { [chain: string]: SourceChainConfig };
 } = {
-  usdc: {
+  uusdc: {
+    avalanche: {
+      id: IS_TESTNET
+        ? ("Avalanche FUJI C-Chain" as const)
+        : ("Avalanche" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0x57F1c63497AEe0bE305B8852b354CEc793da43bB"
+        : "0xfaB550568C688d5D8A52C7d794cb93Edc26eC0eC",
+      logoUrl: "/networks/avalanche.svg",
+    },
+    binance: {
+      id: IS_TESTNET
+        ? ("Binance Smart Chain Testnet" as const)
+        : ("Binance Smart Chain" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0xc2fA98faB811B785b81c64Ac875b31CC9E40F9D2"
+        : "0x4268B8F0B87b6Eae5d897996E6b845ddbD99Adf3",
+      logoUrl: "/networks/binance.svg",
+    },
     ethereum: {
       id: IS_TESTNET
         ? ("Ropsten Test Network" as const)
         : ("Ethereum" as const),
       erc20ContractAddress: IS_TESTNET
         ? "0x526f0A95EDC3DF4CBDB7bb37d4F7Ed451dB8e369"
-        : "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", // test: 'aUSDC' on metamask/etherscan
+        : "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      logoUrl: "/networks/ethereum.svg",
+    },
+    fantom: {
+      id: IS_TESTNET
+        ? ("Fantom Testnet" as const)
+        : ("Fantom" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0x75Cc4fDf1ee3E781C1A3Ee9151D5c6Ce34Cf5C61"
+        : "0x1B6382DBDEa11d97f24495C9A90b7c88469134a4",
+      logoUrl: "/networks/fantom.svg",
+    },
+    moonbeam: {
+      id: IS_TESTNET
+        ? ("Moonbase Alpha TestNet" as const)
+        : ("Moonbeam" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0xD1633F7Fb3d716643125d6415d4177bC36b7186b"
+        : "0xCa01a1D0993565291051daFF390892518ACfAD3A",
+      logoUrl: IS_TESTNET
+        ? "/networks/moonbase.svg"
+        : "/networks/moonbeam.svg",
+    },
+    polygon: {
+      id: IS_TESTNET
+        ? ("Mumbai" as const)
+        : ("Polygon" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0x2c852e740B62308c46DD29B982FBb650D063Bd07"
+        : "0x750e4C4984a9e0f12978eA6742Bc1c5D248f40ed",
+      logoUrl: "/networks/polygon.svg",
+    },
+  },
+  wethwei: {
+    ethereum: {
+      id: IS_TESTNET
+        ? ("Ropsten Test Network" as const)
+        : ("Ethereum" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0xc778417E063141139Fce010982780140Aa0cD5Ab"
+        : "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+      logoUrl: "/networks/ethereum.svg",
+    },
+    avalanche: {
+      id: IS_TESTNET
+        ? ("Avalanche FUJI C-Chain" as const)
+        : ("Avalanche" as const),
+      erc20ContractAddress: IS_TESTNET
+        ? "0x3613C187b3eF813619A25322595bA5E297E4C08a"
+        : "",
```

Comment:
Can we add production contracts too?

Reply:
It was left out because the data is not available. I take it that this can only be done in Testnet so far.